lms/envs/common.py

```diff
@@ -2283,6 +2283,7 @@
 ################# Student Verification #################
 VERIFY_STUDENT = {
     "DAYS_GOOD_FOR": 365,  # How many days is a verficiation good for?
+    "VERIFICATION_EXPIRATION_DAYS": 28,  # How many days before we show message to reverify?
```

Comment:
@rlucioni Do I need to add this setting variable in any other setting variable to work on sandboxes.